libsrc/common/ntohs.s

```diff
@@ -0,0 +1,18 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2023-09-06
+;
+; int __fastcall__ ntohs (int val);
+;
+
+.export     _ntohs, _htons
+.importzp   tmp1
+
+_htons := _ntohs
+
+.code
```

Comment:
The predominant style is to leverage the fact that the assembler defaults to `.code`.